src/widgets/NftDetailWidget/subcomponents/NftDetailAttributes/NftDetailAttributes.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react';
+
+import { Attribute } from '../../../../redux/stores/nftDetail/nftDetailSlice';
```

Comment:
This has to come from `entites/CollectionToken`. But that part @filipmanole is still working on, so it's ok to leave it for now.